src/exprestive.coffee

```diff
@@ -93,22 +93,39 @@ class Exprestive
       url.replace ':id', id
 
 
-  # A helper method for automatically binding restful controllers in a routes file
-  # This is passed as "resources" to the routes file function parameter hash
-  resourcesDirective: (controllerName) =>
+  # Full list of resource action names. It is important that 'new' precede 'show'
+  resourceActions:
+    ['index', 'new', 'show', 'edit', 'update', 'create', 'destroy']
+
+
+  # Returns an object mapping each resource route name to a function which binds the route
+  resourceMappings: (controllerName) ->
     {GET, POST, PUT, DELETE} = @getRouteDirectives()
     singularName = singularize controllerName
     pluralName = pluralize controllerName
 
-    GET "/#{controllerName}",          to: "#{controllerName}#index", as: pluralName
-    GET "/#{controllerName}/new",      to: "#{controllerName}#new",   as: "new_#{singularName}"
-    GET "/#{controllerName}/:id",      to: "#{controllerName}#show",  as: singularName
-    GET "/#{controllerName}/:id/edit", to: "#{controllerName}#edit",  as: "edit_#{singularName}"
-    PUT "/#{controllerName}/:id",      to: "#{controllerName}#update"
-    POST   "/#{controllerName}",       to: "#{controllerName}#create"
-    DELETE "/#{controllerName}/:id",   to: "#{controllerName}#destroy"
+    index:   -> GET "/#{controllerName}",          to: "#{controllerName}#index",   as: pluralName
+    new:     -> GET "/#{controllerName}/new",      to: "#{controllerName}#new",     as: "new_#{singularName}"
+    show:    -> GET "/#{controllerName}/:id",      to: "#{controllerName}#show",    as: singularName
+    edit:    -> GET "/#{controllerName}/:id/edit", to: "#{controllerName}#edit",    as: "edit_#{singularName}"
+    update:  -> PUT "/#{controllerName}/:id",      to: "#{controllerName}#update",  as: singularName
+    create:  -> POST "/#{controllerName}",         to: "#{controllerName}#create",  as: pluralName
+    destroy: -> DELETE "/#{controllerName}/:id",   to: "#{controllerName}#destroy", as: singularName
```

Comment:
It's your call. I put them in so that the paths object would still be populated if you did
```coffeescript
resources 'users', only: ['update', 'create', 'destroy']
```
or any other combo that doesn't include `index` and `show`.